src/renderer/init/Core.js

```diff
@@ -110,6 +159,27 @@ export function initCore(store) {
     store.dispatch(setKeyData(data.mnemonic, data.hdPath, data.privateKeys));
   });
 
+  // The server will send a second message that sets the mnemonic and hdpath
+  ipcRenderer.on(FILECOIN_SET_KEY_DATA, (event, data) => {
+    store.dispatch(filecoinSetKeyData(data.seed, data.privateKeys));
+  });
+
+  ipcRenderer.on(SET_IPFS_URL, (event, data) => {
+    store.dispatch(setIPFSUrl(data.url));
+  });
+
+  ipcRenderer.on(SET_LOTUS_SCHEMA, (event, data) => {
+    store.dispatch(setLotusSchema(data.schema));
+  });
+
+  ipcRenderer.on(SET_CURRENT_OPTIONS, (event, data) => {
+    store.dispatch(setCurrentOptions(data.options));
+  });
+
+  ipcRenderer.on(SET_STORAGE_DEAL_STATUS_ENUM, (event, data) => {
+    store.dispatch(setStorageDealStatusEnum(data.StorageDealStatus));
+  });
+
   ipcRenderer.on(SHOW_HOME_SCREEN, () => {
     store.dispatch(showHomeScreen());
   });
```

Comment:
Can you separate these into sections: Common, Ethereum, Filecoin? With very large comment separators. :)